pkg/webhookconfig/registration.go

```diff
@@ -418,6 +429,13 @@ func (wrc *Register) checkEndpoint() error {
 	if err != nil {
 		return fmt.Errorf("failed to get endpoint %s/%s: %v", config.KyvernoNamespace(), config.KyvernoServiceName(), err)
 	}
+	deploy, err := wrc.GetKubePolicyDeployment()
+	if err != nil {
+		return err
+	}
+	if tlsutils.IsKyvernoInRollingUpdate(deploy) {
+		return errors.New("kyverno is in rolling update")
```

Comment:
Can we inform users how to increase the timeout for webhook registration, by setting the flag?

Reply:
Done